src/Arcus.Templates.WebApi/ExampleProviders/HealthReportExampleProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Swashbuckle.AspNetCore.Filters;
+
+namespace Arcus.Templates.WebApi.ExampleProviders
+{
+    public class HealthReportExampleProvider : IExamplesProvider<HealthReport>
```

Comment:
Some additional XML comments would help here. Make sure that you describe what this providers does, why it's here...